src/Symfony/Bundle/FrameworkBundle/Command/AssetsInstallCommand.php

```diff
@@ -122,7 +122,6 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         // Create the bundles directory otherwise symlink will fail.
         $bundlesDir = $targetArg.'/bundles/';
-        $this->filesystem->mkdir($bundlesDir, 0777);
```

Comment:
given symlink() creates the parent directory, and mkdir() handles parents recursively, we could drop this line.